synapse/rest/admin/rooms.py

```diff
@@ -231,6 +231,31 @@ async def on_GET(self, request, room_id):
         return 200, ret
 
 
+class RoomMembersRestServlet(RestServlet):
+    """
+    Get members list of a room.
+    """
+
+    PATTERNS = admin_patterns("/rooms/(?P<room_id>[^/]+)/members")
+
+    def __init__(self, hs):
+        self.hs = hs
+        self.auth = hs.get_auth()
+        self.store = hs.get_datastore()
+
+    async def on_GET(self, request, room_id):
+        await assert_requester_is_admin(self.auth, request)
+
+        members = await self.store.get_users_in_room(room_id)
+        if not members:
+            print(members)
```

Comment:
Looks like a `print` snuck in here!

Reply:
ah sorry... was testing if I could distinguish empty rooms from not found